Dockerfile

```diff
@@ -45,9 +45,9 @@ RUN if [ -z "$SKIP_FRONTEND_BUILD" ] ; then \
 ADD frontend/package.json /srv/frontend/package.json
 ADD frontend/package-lock.json /srv/frontend/package-lock.json
 WORKDIR /srv/frontend
-RUN mkdir node_modules
 RUN if [ -z "$SKIP_FRONTEND_BUILD" ] ; then \
-    	CI=true npm ci --loglevel warn \
+			npm i -g pnpm && \
```

Comment:
I  think we can use https://nodejs.org/api/corepack.html for activate pnpm
`corepack pnpm install`

Reply:
Hm, I think its too early for `corepack`. It was introduced in v18 and it's experimental.
But worth considering when we move towards update to node18